distribution/debian/install.sh

```diff
@@ -16,51 +17,136 @@
-    echo "You should not run this script from the intended install directory. The script will exit. Please re-run it from another directory"
+    echo ""
+    echo -e "${red}Error!${reset} You should not run this script from the intended install directory."
+    echo "Please re-run it from another directory."
+    echo "Exiting Script!"
     exit
 fi
 
+# User warning about permission conflicts
+echo ""
+echo -e ${red}"           WARNING! WARNING! WARNING!"${reset}
+echo ""
+echo -e "         It is ${red}CRITICAL${reset} that the ${brown}User${reset} and ${brown}Group${reset} you select"
+echo -e "       to run ${brown}[${app^}]${reset} will have both ${red}READ${reset} and ${red}WRITE${reset} access"
+echo -e "      to your Media Library and Download Client directories!"${reset}
+
 # Prompt User
-read -r -p "What user should ${app^} run as? (Default: $app): " app_uid < /dev/tty
+echo ""
+read -r -p "What user should [${app^}] run as? (Default: $app): " app_uid < /dev/tty
 app_uid=$(echo "$app_uid" | tr -d ' ')
 app_uid=${app_uid:-$app}
+
 # Prompt Group
-read -r -p "What group should ${app^} run as? (Default: media): " app_guid < /dev/tty
+echo ""
+read -r -p "What group should [${app^}] run as? (Default: media): " app_guid < /dev/tty
 app_guid=$(echo "$app_guid" | tr -d ' ')
 app_guid=${app_guid:-media}
 
-echo "This will install [${app^}] to [$bindir] and use [$datadir] for the AppData Directory"
-echo "${app^} will run as the user [$app_uid] and group [$app_guid]. By continuing, you've confirmed that that user and group will have READ and WRITE access to your Media Library and Download Client Completed Download directories"
-read -n 1 -r -s -p $'Press enter to continue or ctrl+c to exit...\n' < /dev/tty
+# Info for the user and user confirmation
+echo ""
+echo -e "${brown}[${app^}]${reset} will be installed to ${brown}[$bindir]${reset} and use ${brown}[$datadir]${reset} for the AppData Directory"
+echo ""
+echo -e "${brown}${app^}${reset} will run as the user ${brown}[$app_uid]${reset} and group ${brown}[$app_guid]${reset}."
+echo ""
+echo -e "   By continuing, you ${red}CONFIRM${reset} that that ${brown}[$app_uid]${reset} and ${brown}[$app_guid]${reset}"
+echo -e "   will have both ${red}READ${reset} and ${red}WRITE${reset} access to all required directories."
+
+# User confirmation for installation to continue.
+# User confirmation for installation to continue.
+echo ""
+while true; do
+    read -r -p "Please type 'yes' to continue with the installation: " response
+    response_lowercase=$(echo "$response" | tr '[:upper:]' '[:lower:]')
+    if [[ $response_lowercase == "yes" ]]; then
+        break
+    elif [[ $response_lowercase == "y" ]]; then
+        echo ""
+        echo "You must type in 'y e s' to continue with installation."
+        echo ""
+        continue
+    else
+        echo ""
+        echo "Invalid response! Operation is canceled!"
+        echo "Exiting script!"
+        exit 0
+    fi
+done
+
```

Comment:
Why require `yes`, if you've identified the first letter is `y` then I'd let it pass.